setup.py

```diff
@@ -158,7 +135,6 @@
     "requests",
     "rhoknp>=1.1.0,<1.3.1",
     "rjieba",
-    "rouge-score!=0.0.7,!=0.0.8,!=0.1,!=0.1.1",
```

Comment:
no references to `rouge-score` (library name) nor `rouge_score` (import name) in our library